hack/local-up-cluster.sh

```diff
@@ -27,6 +27,8 @@ ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-""}
 PSP_ADMISSION=${PSP_ADMISSION:-""}
 NODE_ADMISSION=${NODE_ADMISSION:-""}
 RUNTIME_CONFIG=${RUNTIME_CONFIG:-""}
+# RUNTIME_CONFIG=${RUNTIME_CONFIG:-"settingsio/v1alpha1=true,settings.k8s.io/v1alpha1/podpreset=true"}
+RUNTIME_CONFIG=${RUNTIME_CONFIG:-"api/all=true"}
```

Comment:
What's your intended goal for "RUNTIME_CONFIG"? You're attempting to declare it on both lines 29 and 31.